commit b36db9f663ea0828d81245314e9f7465c751cd85
Author: Raphael Kubo da Costa <rakuco@FreeBSD.org>
Date:   Sun Jun 23 20:49:17 2013 +0300

    Enable exceptions in kalarmcal/ as well.
    
    clang seems to be more strict than gcc, and complains about exceptions being
    disabled despite akonadi/itempayloadinternals_p.h being indirectly included.

diff --git a/kalarmcal/CMakeLists.txt b/kalarmcal/CMakeLists.txt
index 3f27597..7c6a4fa 100644
--- kalarmcal/CMakeLists.txt
+++ kalarmcal/CMakeLists.txt
@@ -31,6 +31,9 @@ set(kalarmcal_LIB_SRCS
     version.cpp
    )
 if(NOT USE_KRESOURCES)
+    # The includes in akonadi/ use exceptions, we must enable them here for clang to be happy.
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${KDE4_ENABLE_EXCEPTIONS}")
+
     set(kalarmcal_LIB_SRCS ${kalarmcal_LIB_SRCS}
 	collectionattribute.cpp
 	compatibilityattribute.cpp
